validators/migrate-vali/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Migrate Validator"
+description: >
+  How to migrate an Autonity validator node to a new instance or to a new IP/Port address
+---
+
+::: {.callout-note title="Prerequisites" collapse="false"}
+- Ensure that the host machine meets the [minimum requirements](/node-operators/install-aut/#requirements)
+:::
+
+
+## Migrating validator node to a new environment
+
+To migrate a node to a new instance follow the steps described in the node operator's guide [Migrate Autonity](/node-operators/migrate-aut/).
+
+::: {.callout-warning title="Important" collapse="false"}
+If your node is in an active state, you must [pause the validator node](/validators/pause-vali/) **before migration**. [Reactivate the validator](/validators/pause-vali/) **after migration**.
+
+Be sure to fully decommission the original node installation!
+:::
+
+
+## Migrating validator node to a new IP/Port address
+
+This guide describes how to migrate a validator node to an environment with a new IP/Port network address by updating the validator's registered enode URL.
+
+::: {.callout-warning title="Important" collapse="false"}
+Only the network connection information - IP and/or port - of a registered validator enode can be updated. You **cannot** change the validator's address (i.e. the `PUBKEY` part of the [validator enode URL](/concepts/validator/#validator-enode-url)).
+:::
+
+When performing the update the validator **must not** not be an active member of the consensus committee and should be in a [`paused`](/concepts/validator/#validator-pausing) state.
```

Comment:
Only requirement is to "must noe be an active member of the consensus committee". Paused state is one of the way to achieve that but not a requirement.

Reply:
Yes. I have reworded this section so:

>When performing the update the validator **must not** not be an active member of the consensus committee. It is **recommended** that you [`pause`](/concepts/validator/#validator-pausing) your validator before migration. This is to prevent your node being elected as a committee member while you are in the process of migration.

Commit: 49f78998a49933052a54dcec360996d63e0413b5.